packages/ffmpeg/src/classes.ts

```diff
@@ -161,17 +168,24 @@ export class FFmpeg {
    * @category FFmpeg
    * @returns `true` if ffmpeg core is loaded for the first time.
    */
-  public load = (config: FFMessageLoadConfig = {}): Promise<IsFirst> => {
+  public load = (
+    config: FFMessageLoadConfig = {},
+    { signal }: FFMessageOptions = {}
```

Comment:
for every methods using #send : add an options param to pass the optional signal